samples/src/Microsoft.Azure.IIoT.App/src/Pages/PublishedNodes.razor

```diff
@@ -28,59 +30,69 @@
-        {
+        @foreach (var node in PagedNodeList.Results) {
             <tr>
                 <td class="hover-text width-large">
                     <div>
                         @{ string NodeId = CommonHelper.None;
-                            NodeId = string.IsNullOrEmpty(node.NodeId) ? CommonHelper.None : node.NodeId;
+                            NodeId = string.IsNullOrEmpty(node.PublishedItem.NodeId) ? CommonHelper.None : node.PublishedItem.NodeId;
                         } @NodeId
                     </div>
                 </td>
-                <td class="hover-text width-large">
+                <td class="hover-text width-medium-large">
                     <div>
                         @{ string DisplayName = CommonHelper.None;
-                            DisplayName = string.IsNullOrEmpty(node.DisplayName) ? CommonHelper.None : node.DisplayName;
+                            DisplayName = string.IsNullOrEmpty(node.PublishedItem.DisplayName) ? CommonHelper.None : node.PublishedItem.DisplayName;
                         } @DisplayName
                     </div>
                 </td>
                 <td class="hover-text width-small">
                     <div>
                         @{ string PublishingInterval = CommonHelper.None;
-                            PublishingInterval = node.PublishingInterval == null ? CommonHelper.None : node.PublishingInterval.ToString();
+                            PublishingInterval = node.PublishedItem.PublishingInterval == null ? CommonHelper.None : node.PublishedItem.PublishingInterval.ToString();
                         } @PublishingInterval
                     </div>
                 </td>
                 <td class="hover-text width-small">
                     <div>
                         @{ string SamplingInterval = CommonHelper.None;
-                            SamplingInterval = node.SamplingInterval == null ? CommonHelper.None : node.SamplingInterval.ToString();
+                            SamplingInterval = node.PublishedItem.SamplingInterval == null ? CommonHelper.None : node.PublishedItem.SamplingInterval.ToString();
                         } @SamplingInterval
                     </div>
                 </td>
                 <td class="hover-text width-small">
                     <div>
                         @{ string HeartbeatInterval = CommonHelper.None;
-                            HeartbeatInterval = node.HeartbeatInterval == null ? CommonHelper.None : node.HeartbeatInterval.ToString();
+                            HeartbeatInterval = node.PublishedItem.HeartbeatInterval == null ? CommonHelper.None : node.PublishedItem.HeartbeatInterval.ToString();
                         } @HeartbeatInterval
                     </div>
                 </td>
+                <td class="width-medium-large">
+                    <div class="value-scroll">
+                        @{string outputValue = $"{node.Value}";}
```

Comment:
because outputstring is not just node.xxx, but $"{node.xxx}" and it's the easy way to put in a <span> tag

Reply:
Sorry to ask again, maybe I'm missing something, but I still think this can be written in a simpler way:
![image](https://user-images.githubusercontent.com/1353540/84518326-d5e5be80-acd0-11ea-9be3-be206f5b348e.png)

Reply:
I followed your suggestion. The issue was the '$' but is not needed here